extensions/Worker.Extensions.SignalRService/Worker.Extensions.SignalRService.csproj

```diff
@@ -6,7 +6,7 @@
     <Description>Azure SignalR Service extensions for .NET isolated functions</Description>
 
     <!--Version information-->
-    <VersionPrefix>1.2.2</VersionPrefix>
+    <VersionPrefix>1.6.0</VersionPrefix>
 
     <!--Temporarily opting out of documentation. Pending documentation-->
     <GenerateDocumentationFile>false</GenerateDocumentationFile>
```

Comment:
One minor change would be this property. With the XML docs in place, this should be enabled.